src/components/measureGroups/MeasureGroups.tsx

```diff
@@ -504,6 +504,11 @@ const MeasureGroups = () => {
                         renderInput={(params) => (
                           <TextField {...params} label="" />
                         )}
+                        onKeyDown={(e) => {
+                          if (e.keyCode === 13) {
+                            e.preventDefault();
```

Comment:
seems like there have a been a number of these now. Is it maybe time to create a util function to do this? i.e.
`onKeyDown={preventSubmitOnEnter}`
```
export function preventSubmitOnEnter(e) {
  if (e.keyCode === 13) {
    e.preventDefault()
  }
}
```

Reply:
Nate had a look again and he is ok with the change of switching to DSLink.